### PR TITLE
OM Digital Solutions OM-5 camconst #6699

### LINKED ISSUE #6699: Support for OM Digital Solutions OM-5
With the recent merge of initial support for the OMDS OM-1 (2d37dec49b693687008dba8c1ae02f468f180f45), rt can open raw files from OM-5.
Colors are a bit off and white frames show up as pink, so a camconst.json entry needs to be created/derived.

I have taken two sets of white frames for all ISO levels at F5.6 with LENR on (3" exposure) and off (1" exposure).
The files are available [*here*](https://filebin.net/7ql2q8sgsbq9k2ne/OM_Digital_Solutions_OM-5.tgz)

### PR DESCRIPTION
**Pull request by Beep6581**

### CODE REVIEW

File: rtengine/camconst.json [hunk 1]
```diff
@@ -2185,6 +2185,16 @@ Camera constants:
         ]
     },
 
+    { // Quality X
+        "make_model": ["OM Digital Solutions OM-5"],
+        "dcraw_matrix": [ 11896, -5110, -1076, -3181, 11378, 2048, -519, 1224, 5166 ], // ColorMatrix2 using illuminant D65 from Adobe DNG Converter 15.0
+        "raw_crop" : [
+           { "frame" : [8200, 6132], "crop": [0, 0, 8172, 6132] },
+           { "frame" : [5240, 3912], "crop": [0, 0, 5240, 3912] },
+           { "frame" : [10400, 7792], "crop": [0, 0, 10392, 7792] }
```
Comment (Lawrence37) [inline rtengine/camconst.json]: There is still 2 pixels of garbage values on the right border of the 80 MP image.
image

**Comment by Lawrence37** [comment]
Converting this to a draft due to the incorrect raw crop. Adding the white level(s) would also be nice.

**Comment by Lawrence37** [comment]
I removed an extra 2 pixels from the right border of the 80 megapixel raw crop and added the white level based on the averages provided by @michael-lenz in https://github.com/Beep6581/RawTherapee/issues/6699#issuecomment-1454732489 and the recommendation stated in `camconst.json`:
>you set the white level around 50-100 14-bit units below or 10-20 12-bit units

https://github.com/Beep6581/RawTherapee/blob/0404a47fb7c7acd70fcc943a34775f2688f3d52b/rtengine/camconst.json#L184-L187

**Comment by Lawrence37** [comment]
Ok for merge.

**Comment by Beep6581** [comment]
Confirmed incorrect raw crop and fix.

**Pull request merged**